src/core/utils/fs.cpp

```diff
@@ -453,6 +453,18 @@ QString Utils::Fs::QDesktopServicesDownloadLocation()
 #endif
 #endif
 
+#if defined(Q_OS_WIN)
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
+    if (QSysInfo::windowsVersion() <= QSysInfo::WV_XP)  // Windows XP
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).absoluteFilePath(
+                    QCoreApplication::translate("fsutils", "Downloads"));
+    else
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::HomeLocation)).absoluteFilePath("Downloads");
+#else
+    return QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);
```

Comment:
> So does this function still point to the correct folder?

On Vista+, the function is working correctly as davispuh said.
On XP, the `Downloads` folder is using translated string unlike on Vista+. Unfortunately I couldn't compile a binary to run in XP and test the function.

> Then @Chocobo1 can you use...

Will change.